jitbuilder/control/Jit.cpp

```diff
@@ -179,7 +179,7 @@ extern "C"
 bool
 initializeJit()
    {
-   return initializeJitBuilder(0, 0, 0, (char *)"-Xjit:acceptHugeMethods,enableBasicBlockHoisting,omitFramePointer");
+   return initializeJitBuilder(0, 0, 0, (char *)"-Xjit:acceptHugeMethods,enableBasicBlockHoisting,omitFramePointer,useIlValidator");
```

Comment:
Do we want the validator turned on by default for JitBuilder? That's a bit surprising to me.